IPython/core/crashhandler.py

```diff
@@ -53,6 +53,16 @@
 If you want to do it now, the following command will work (under Unix):
 mail -s '{app_name} Crash Report' {contact_email} < {crash_report_fname}
 
+Please also include in your email:
+- information about your operating system: is it Linux, macOS, Windows, other,
+  and which version, for example Ubuntu 16.04, macOS 10.13.2, Windows 10 Pro,
+  is it 32-bit or 64-bit, ...
+- information on how you installed {app_name}: from binary, compiled from source,
+  via Docker, on a virtual machine...
+- information on how to reproduce the crash: what exact sequence of instructions
```

Comment:
Since all three points start with "information on", let's phrase the bit at the top "Please also include information on:" and cut those words from the bullet points.

Also, I'd like the bullet points to start with capital letters  ("How to reproduce...").